en/components/accordion.md

```diff
@@ -0,0 +1,264 @@
+           data-demos-base-url="{environment:demosBaseUrl}" 
+           iframe-src="{environment:demosBaseUrl}/layouts/accordion-sample-1" alt="Angular Avatar Example">
+</code-view>
+
+<div class="divider--half"></div>
+
+## Getting Started with Ignite UI for Angular Accordion
+
+To get started with the Ignite UI for Angular Accordion component, first you need to install Ignite UI for Angular. In an existing Angular application, type the following command:
+
+```cmd
+ng add igniteui-angular
+```
+For a complete introduction to the Ignite UI for Angular, read the [*getting started*](general/getting-started.md) topic.
+
+The next step is to import the `IgxAccordionModule` in your app.module file. 
+
+```typescript
+// app.module.ts
+
+...
+import { IgxAccordionModule } from 'igniteui-angular';
+
+@NgModule({
+    ...
+    imports: [..., IgxAccordionModule],
+    ...
+})
+export class AppModule {}
+```
+Now that we have the accordion module imported, let’s get started with a basic configuration of the `igx-accordion` and its panels.
+
+## Usage
+
+Every separate section in the [IgxAccordionComponent]({environment:angularApiUrl}/classes/igxaccordioncomponent.html) is defined with an [expansion panel]({environment:angularApiUrl}/classes/igxexpansionpanelcomponent.html).
+Panels provide [disabled]({environment:angularApiUrl}/classes/igxexpansionpanelcomponent.html#disabled), [collapsed]({environment:angularApiUrl}/classes/igxexpansionpanelcomponent.html#collapsed) and [animationSettings]({environment:angularApiUrl}/classes/igxexpansionpanelcomponent.html#animationSettings}) properties, which give you the opportunity to configure the states of the panel as per your requirement. 
+
+### Declaring an accordion
+
+The accordion simply renders any [`igx-expansion-panels`]({environment:angularApiUrl}/classes/igxexpansionpanelcomponent.html) declared inside it.
+
+```html
+<igx-accordion #accordion [singleBranchExpand]="true">
+     <igx-expansion-panel>
+        <igx-expansion-panel-header>
+            <igx-expansion-panel-title>Title Panel 1</igx-expansion-panel-title>
+        </igx-expansion-panel-header>
+        <igx-expansion-panel-body>
+            Content Panel 1
+        </igx-expansion-panel-body>
+    </igx-expansion-panel>
+    <igx-expansion-panel>
+        <igx-expansion-panel-header>
+            <igx-expansion-panel-title>Title Panel 2</igx-expansion-panel-title>
+        </igx-expansion-panel-header>
+        <igx-expansion-panel-body>
+            Content Panel 2
+        </igx-expansion-panel-body>
+    </igx-expansion-panel>
+</igx-accordion>
+```
+
+Using the [panels]({environment:angularApiUrl}/classes/igxaccordioncomponent.html#panels) accessor you can get a reference to the collection containing all [expansion panels]({environment:angularApiUrl}/classes/igxexpansionpanelcomponent.html) children of the `igx-accordion`.
+
+```typescript
+@ViewChild('accordion', { static: true })
+public accordion!: IgxAccordionComponent;
```

Comment:
```suggestion
public accordion: IgxAccordionComponent;
```

Reply:
@kdinev 
When typescript compiler options have `strict` enabled, it leads to an error when typing properties that aren't initialized - [strict class initialization](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-7.html#strict-class-initialization). Since `strict` is enabled by default in @angular/cli 12+ projects, I figured we should try and keep that in mind for newer code snippets.